test/scales/linearScaleTests.ts

```diff
@@ -118,6 +118,29 @@ describe("Scales", () => {
       assert.deepEqual(scale.domain(), [-1, 5], "Regular domains still accepted");
     });
 
+    describe("domain snapping", () => {
+      it("domain snapping setter and getter", () => {
+        let scale = new Plottable.Scales.Linear();
+
+        assert.strictEqual(scale.snapsDomain(), true, "scales make their domain snap by default");
+        assert.strictEqual(scale.snapsDomain(false), scale, "setting disabling domain snapping returns the scale");
+        assert.strictEqual(scale.snapsDomain(), false, "the domain is no longer snaps");
+      });
+
+      it("domain snapping works", () => {
+        let scale = new Plottable.Scales.Linear();
+        scale.addIncludedValuesProvider(function() {
+          return [1.123123123, 3.123123123];
+        });
+
+        assert.deepEqual(scale.domain(), [1, 3.2], "domain snapping works");
+        scale.snapsDomain(false);
+        assert.deepEqual(scale.domain(), [1.073123123, 3.173123123], "domain snapping can be deactivated");
```

Comment:
Rather than using magic numbers here, can you set `paddingProportion(0)`, then assert that the domain matches the included values?